src/server.c

```diff
@@ -1179,21 +1071,41 @@ void databasesCron(void) {
          * DB we'll be able to start from the successive in the next
          * cron loop iteration. */
         static unsigned int resize_db = 0;
+        static unsigned int rehash_db = 0;
         int dbs_per_call = CRON_DBS_PER_CALL;
         int j;
 
         /* Don't test more DBs than we have. */
         if (dbs_per_call > server.dbnum) dbs_per_call = server.dbnum;
 
-        /* Resize */
+        /* In cluster-enabled setup, this method traverses through all main/expires
+         * dictionaries (CLUSTER_SLOTS) and triggers a resize if the percentage of
+         * used buckets in the HT reaches (100 / HASHTABLE_MIN_FILL).
+         * We shrink the hash table to save memory, or expand the hash when the
+         * percentage of used buckets reached 100.
+         *
+         * In non cluster-enabled setup, it resize main/expires dictionary based on
+         * the same condition described above. */
```

Comment:
i think this comment is outdated
also, maybe part of it now belongs in kvstoreTryResizeDicts?